open_schools_platform/organization_management/organizations/views.py

```diff
@@ -151,8 +151,8 @@ class FilterProperties:
 
     queryset = Query.objects.all()
     swagger_filter_fields = \
-        FilterProperties.query_fields | \
-        FilterProperties.student_fields | \
+        FilterProperties.query_fields or \
+        FilterProperties.student_fields or \
```

Comment:
?

Reply:
Yes, forgot to remove it, working on it right now (because I have strange error with "|" symbol)

Reply:
you still have errors with |? Try to use higher python version

Reply:
Thx for advise!